samples: accel_trig: mimxrt1040_evk: twr_ke18f: trigger mode will cause build error

**Describe the bug**

accel_trig build failure due the enable trigger mode. as the trigger gpio need rework board, and is disabled by dts by default.  

Please also mention any information which could help others to understand
the problem you're facing:
 - What target platform are you using?
 - mimxrt1040_evk
 - twr_ke18f
 - What have you tried to diagnose or workaround this issue?
 - code analysis
 - Is this a regression? If yes, have you been able to "git bisect" it to a
   specific commit?

```
d40eed43953550333d9b265358c85b5dd2d590a6 is the first bad commit
commit d40eed43953550333d9b265358c85b5dd2d590a6
Author: Hake Huang <hake.huang@oss.nxp.com>
Date:   Thu Dec 12 17:01:03 2024 +0800

    samples: sensor: enable sensor trigger

    1. enable sensor trigger.
    2. add a fixture for cases, as some platform need
       special rework.

    Signed-off-by: Hake Huang <hake.huang@oss.nxp.com>

```

**To Reproduce**

```
scripts/twister -p mimxrt1040_evk -T samples/sensor/accel_trig
```

**Expected behavior**

build pass

**Impact**

CI

**Logs and console output**

```
/home/jenkins/agent/workspace/zephyr_downstream_build_daily/zephyr/include/zephyr/device.h:96:41: error: '__device_dts_ord_DT_N_S_soc_S_i2c_403f8000_S_fxls8974_18_P_int2_gpios_IDX_0_PH_ORD' undeclared here (not in a function) 96 | #define DEVICE_NAME_GET(dev_id) _CONCAT(__device_, dev_id) | ^~~~~~~~~ /home/jenkins/agent/workspace/zephyr_downstream_build_daily/zephyr/include/zephyr/sys/util_internal.h:72:26: note: in definition of macro '__DEBRACKET' 72 | #define __DEBRACKET(...) __VA_ARGS__ | ^~~~~~~~~~~ /home/jenkins/agent/workspace/zephyr_downstream_build_daily/zephyr/include
```

**Environment (please complete the following information):**

 - OS: (e.g. Linux, MacOS, Windows)
 - Toolchain (e.g Zephyr SDK, ...)
 - Commit SHA or Version used: nxp-v4.0.0-12248-g638154ac8a5e

--- a/samples/sensor/accel_trig/boards/mimxrt1040_evk.overlay
+++ b/samples/sensor/accel_trig/boards/mimxrt1040_evk.overlay
@@ -1,0 +1,12 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * Copyright 2025 NXP
+ */
+
+
+&fxls8974 {
+	status = "okay";
+	int1-gpios = <&gpio1 11 GPIO_ACTIVE_LOW>;
+	int2-gpios = <&gpio1 12 GPIO_ACTIVE_LOW>;
+};

--- a/samples/sensor/accel_trig/boards/twr_ke18f.overlay
+++ b/samples/sensor/accel_trig/boards/twr_ke18f.overlay
@@ -1,0 +1,11 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * Copyright 2025 NXP
+ */
+
+&fxos8700 {
+	status = "okay";
+	int1-gpios = <&gpioa 14 GPIO_ACTIVE_LOW>;
+	int2-gpios = <&gpioa 17 GPIO_ACTIVE_LOW>;
+};

--- a/samples/sensor/magn_trig/boards/twr_ke18f.overlay
+++ b/samples/sensor/magn_trig/boards/twr_ke18f.overlay
@@ -1,0 +1,12 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * Copyright 2025 NXP
+ */
+
+
+&fxos8700 {
+	status = "okay";
+	int1-gpios = <&gpioa 14 GPIO_ACTIVE_LOW>;
+	int2-gpios = <&gpioa 17 GPIO_ACTIVE_LOW>;
+};

--- a/samples/sensor/magn_trig/sample.yaml
+++ b/samples/sensor/magn_trig/sample.yaml
@@ -6,7 +6,7 @@ tests:
     tags: sensors
     filter: dt_alias_exists("magn0")
     harness_config:
-      fixture: fixture_sensor_accel_int
+      fixture: fixture_sensor_magn_int
       type: one_line
       regex:
         - "^\\s*[0-9A-Za-z_,+-.]*@[0-9A-Fa-f]* \\(x, y, z\\):    \

